<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowTester</class>
 <widget class="QMainWindow" name="MainWindowTester">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Redmine [режим тестировщика]</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icon</normaloff>:/icon</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLineEdit" name="editFind">
      <property name="inputMask">
       <string>00000</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_4" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="MainWindowTesterSplitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTableWidget" name="tableTest">
             <property name="editTriggers">
              <set>QAbstractItemView::EditKeyPressed</set>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTableWidget" name="tableTask">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="editName">
             <property name="toolTip">
              <string>Название задачи</string>
             </property>
             <property name="maxLength">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboProject">
             <property name="toolTip">
              <string>Алгоритм</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboTrack"/>
           </item>
           <item>
            <widget class="QWidget" name="widget_5" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QComboBox" name="comboLevel"/>
              </item>
              <item>
               <widget class="QPushButton" name="btLevelDefault">
                <property name="maximumSize">
                 <size>
                  <width>24</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>По умолчанию</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="icons.qrc">
                  <normaloff>:/ok</normaloff>:/ok</iconset>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboUser">
             <property name="toolTip">
              <string>Исполнитель</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="editComment">
             <property name="toolTip">
              <string>Подробное описание задачи</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widgetButtons" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="btCancelTask">
                <property name="palette">
                 <palette>
                  <active>
                   <colorrole role="Button">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>207</red>
                      <green>255</green>
                      <blue>207</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </active>
                  <inactive>
                   <colorrole role="Button">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>207</red>
                      <green>255</green>
                      <blue>207</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </inactive>
                  <disabled>
                   <colorrole role="Button">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>207</red>
                      <green>255</green>
                      <blue>207</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </disabled>
                 </palette>
                </property>
                <property name="toolTip">
                 <string>Если задача выполнена и все корректно работает, жми эту кнопку!</string>
                </property>
                <property name="text">
                 <string>Все работает</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btReworkingTask">
                <property name="palette">
                 <palette>
                  <active>
                   <colorrole role="Button">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>255</red>
                      <green>207</green>
                      <blue>207</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </active>
                  <inactive>
                   <colorrole role="Button">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>255</red>
                      <green>207</green>
                      <blue>207</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </inactive>
                  <disabled>
                   <colorrole role="Button">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>255</red>
                      <green>207</green>
                      <blue>207</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </disabled>
                 </palette>
                </property>
                <property name="toolTip">
                 <string>Если что-то не работает, жми эту кнопку!</string>
                </property>
                <property name="text">
                 <string>Не работает</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUpdate"/>
   <addaction name="actionSettings"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionSettings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/settings</normaloff>:/settings</iconset>
   </property>
   <property name="text">
    <string>Настройки</string>
   </property>
   <property name="toolTip">
    <string>Настройки</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/update</normaloff>:/update</iconset>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
   <property name="toolTip">
    <string>Обновить</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/exit</normaloff>:/exit</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="toolTip">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
